src/specific/s_clock.h

```diff
@@ -1,5 +1,5 @@
-#ifndef T1M_SPECIFIC_CLOCK_H
-#define T1M_SPECIFIC_CLOCK_H
+#ifndef T1M_SPECIFIC_S_CLOCK_H
+#define T1M_SPECIFIC_S_CLOCK_H
```

Comment:
Not a huge deal to me, but the `SPECIFIC` and `S` are redundant. Just double checking if this was supposed to change to `_S_` only or not.

Reply:
True – it is redundant – but I wanted this scheme to be very simple and have no exceptions.

Reply:
Ya that works.